doc/doxygen/user_manual/running_apps/running_apps.md

```diff
@@ -0,0 +1,134 @@
+- anisotropyFacet: A different strong anisotropy formation than in CHAC\_anisotropyRegularized that is easier to specify particular facets in the Wulff shape. (2D)
+- steadyStateAllenCahn: An implementation of coupled Allen-Cahn and steady-state Allen-Cahn equations as a demonstration of the nonlinear solver. (2D)
+- fickianDiffusion: An implementation of the diffusion equation with a time-dependent source term. (2D)
+- mechanics: An implementation of linear elasticity for a material in uniaxial tension. (3D)
+- CHiMaD\_benchmark1a: An implementation of the CHiMaD spinodal decomposition benchmark problem. (2D)
+- CHiMaD\_benchmark2a: An implementation of the CHiMaD Ostwald ripening benchmark problem. (2D)
+- CHiMaD\_benchmark3: An implementation of the CHiMaD dendritic solidification benchmark problem. (2D)
+- CHiMaD\_benchmark6a: An implementation of the CHiMaD electrochemistry benchmark problem. (2D)
+- CHiMaD\_benchmark6b: An implementation of the CHiMaD electrochemistry benchmark problem with a curved domain. (2D)
+- dendriticSolidification: An implementation of a solidification model for a pure material resulting in the growth of a dendrite. (2D)
+- eshelbyInclusion: An implementation of linear elasticity for a spherical inclusion. (3D)
+- grainGrowth: An implementation of coupled Allen-Cahn equations simulating grain growth in two dimensions. (2D)
+- grainGrowth_dream3D: An implementation of coupled Allen-Cahn equations simulating grain growth in two dimensions with an initial microstructure imported from Dream3D. (2D)
+- precipiateEvolution: An implementation of the coupled Cahn-Hilliard/Allen-Cahn/Linear Elasticity equations often used in phase field simulation of precipitate evolution. (2D)
+- precipiateEvolution_pfunction: Like precipitateEvolution, but loads inputs using PRISMS IntegrationTools. (2D)
+- MgNd_precipitate_single_Bppp: Similar to precipiateEvolution, but uses the KKS model rather than the WBM model for the free energy functional. The parameters are set for \f$\beta\f$''' precipitates in an Mg-Nd alloy from [this publication](https://www.sciencedirect.com/science/article/pii/S1359645417305281). (3D)
+- nucleationModel: KKS precipitation model that makes use of the PRISMS-PF explicit nucleation capabilities. (2D)
+- nucleationModel_preferential: Like nucleationModel, but with a zone with an increased nucleation rate to simulate a grain boundary. (2D)
+
+A directory for each of these apps can be found in the [applications directory](https://github.com/prisms-center/phaseField/tree/master/applications) (i.e. phaseField/applications). The apps contain a formulation file giving the governing equations. In addition to the 24 apps listed above, some app names may be preceded by an underscore. The underscore is used to denote apps that are still under active development.
+
+## Compiling and Running the Allen-Cahn Example App
```

Comment:
At some point before this, hyperlink to install_prismspf

Reply:
I haven't visualized the site yet. Im guessing this already exists in a sidebar.